src/main/resources/errors.yaml

```diff
@@ -805,9 +815,11 @@ request-errors:
     body: |-
       The command attempted to create a map column that used unsupported types for either the key or value.
       
-      Maps can only use `text` or `ascii` types for the key type, and any primitive type for the value.
+      Maps can only use primitive type for the key and value with following exceptions:
+      1. The key can't be duration, counter, timeuuid.
```

Comment:
ok for now, if we open this up again it would be better to have the code supply the lists of supported types , but no need to change now.